tests/unit/utils/txn_parser/test_parse_balance_changes.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+from unittest import TestCase
+
+from tests.unit.utils.txn_parser.transaction import TXN
```

Comment:
nit: this transaction should probably be named something a little more descriptive than `TXN`